Counties/Florida/Bay County/Scraper/utils/ScraperUtils.py

```diff
@@ -61,40 +61,78 @@ def parse_plea_case_numbers(plea_text: str, valid_charges: List[int]) -> List[in
     :param valid_charges: Valid charge numbers that can be referenced in the plea text
     :return: List of charge count numbers related to this plea
     """
-    plea_text = plea_text.strip().split()
-    if len(plea_text) > 0:
-        plea_counts = re.sub('[^,0-9]', '', plea_text[-1]).split(',')
-        plea_number = [int(charge) for charge in plea_counts if charge != '']
-        # In rare cases numbers appear in the plea text not relating to a count number. Filter these out.
-        filtered_plea_number = [charge for charge in plea_number if charge in valid_charges]
-        return filtered_plea_number
-    else:
+    try:
+        plea_text = plea_text.strip().split()
+        if len(plea_text) > 0:
+            plea_counts = re.sub('[^,0-9]', '', plea_text[-1]).split(',')
+            plea_number = [int(charge) for charge in plea_counts if charge != '']
+            # In rare cases numbers appear in the plea text not relating to a count number. Filter these out.
+            filtered_plea_number = [charge for charge in plea_number if charge in valid_charges]
+            return filtered_plea_number
+        else:
+            return []
+    except (AttributeError, TypeError):
         return []
 
 
+def parse_charge_statute(charge_text: str) -> (str, str):
+    """
+    Parses the charge description text into the charge and statute code separately
+    :param charge_text: Charge description
+    :return: (Charge description, Statute code)
+    """
+    try:
+        charge_text = charge_text.strip()
+    except AttributeError:
+        return None, None
+
+    # Find last set of parenthesis, see https://stackoverflow.com/a/42147313/6008271
+    try:
+        statute = re.search("\(([^()]*)\)$", charge_text, re.IGNORECASE).group(1)
```

Comment:
You *should* be able to replace this with:
```
match = re.search("\(([^()]*)\)$", charge_text, re.IGNORECASE)
statute = match.group(1) if match else None
```
This would bypass the need for the try/except block.